src/components/sidebar/routePathView/routePathListTab/RoutePathListLink.tsx

```diff
@@ -86,91 +167,49 @@ class RoutePathListLink extends React.Component<IRoutePathListLinkProps> {
                 </div>
                 <div className={s.flexRow}>
-                    <div className={s.flexInnerRow}>
-                        <input
-                            placeholder='1'
-                            type='text'
-                            className={s.smallInput}
-                        />
-                        <Checkbox
-                            checked={false}
-                            content='Ohitusaika kirja-aikat.'
-                            onClick={this.onChange}
-                        />
-                    </div>
-                    <div className={s.flexInnerRow}>
-                        <input
-                            placeholder='1'
-                            type='text'
-                            className={s.smallInput}
-                        />
-                        <Checkbox
-                            checked={false}
-                            content='Ohitusaika nettiaikat.'
-                            onClick={this.onChange}
-                        />
-                    </div>
-                </div>
-                <div className={s.flexRow}>
-                    <div className={s.inputLabel}>
-                        VIIM. LINKIN LOPPUSOLMU SARAKE NRO
-                    </div>
+                    <Checkbox
+                        disabled={isEditingDisabled}
+                        checked={routePathLink.isAtBookSchedule}
+                        content='Laitetaanko ohitusaika kirja-aikatauluun?'
+                        onClick={this.onCheckboxChange(
+                            'isAtBookSchedule',
+                            routePathLink.isAtBookSchedule
+                        )}
+                    />
                 </div>
                 <div className={s.flexRow}>
-                    <div className={s.flexInnerRow}>
-                        <input
-                            placeholder='1'
-                            type='text'
-                            className={s.smallInput}
-                        />
-                        <Checkbox
-                            checked={false}
-                            content='Ohitusaika nettiaikat.'
-                            onClick={this.onChange}
-                        />
-                    </div>
-                    <div className={s.flexInnerRow}>
-                        <input
-                            placeholder='1'
-                            type='text'
-                            className={s.smallInput}
-                        />
-                        <Checkbox
-                            checked={false}
-                            content='Ohitusaika kirja-aikat.'
-                            onClick={this.onChange}
-                        />
-                    </div>
+                    <InputContainer
+                        disabled={isEditingDisabled}
+                        type={'number'}
```

Comment:
No need to wrap string with {}

Reply:
Also found some other places to fix this with ctrl + F